docs/distro-compat.md

```diff
@@ -1,111 +1,30 @@
-  kernel be booted with cgroups v1. To boot the Fedora kernel with cgroups v1, set
-  kernel parameter `systemd.unified_cgroup_hierarchy=0` as follows and reboot the
-  kernel.
-
-  ```console
-  sudo dnf install grubby
-  sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
-  ```
-
-## Sysbox Package
-
-We are currently offering the [Sysbox packages](https://github.com/nestybox/sysbox/releases)
-for Ubuntu and Debian distributions. This means that for other distros you must [build and install Sysbox from source](https://github.com/nestybox/sysbox/blob/master/docs/developers-guide/build.md).
-It's pretty easy though, since the Sysbox Makefile has targets that automate this process.
-The build occurs entirely inside a container, so it won't mess up your host.
-
-We are working on creating packaged versions for the other supported distros, and
-expect to have them soon (ETA summer 2021).
-
-## Sysbox Operational Modes
-
-The Sysbox runtime relies on the kernel's user-namespace feature to secure
-system containers. There are two approaches utilized by Sysbox to manage the
-creation of these user-namespaces: `automatic` and `directed`.
-
-It is important to highlight that this operational mode is per individual
-container. That is, a container-manager (e.g. Docker, K8s/Cri-o, Podman, etc)
-could instruct Sysbox to operate in `directed` mode for any given container.
-
-What follows is a high-level description of what each of these modes accomplish;
-refer to the [user-guide](./user-guide/security.md#user-namespace-id-mapping) for
-more details.
-
-### Automatic userns ID mapping
-
-In this mode Sysbox automatically allocates the ID mappings to be used by the
-container. By doing this, Sysbox enforces strong container isolation without
-requiring the higher level container-manager to do this explicitly (e.g. without
-enabling `userns-remap` mode in Docker).
-
-Sysbox will only attempt to operate in this mode if the received container
-specification omits ID mappings configuration.
-
-Note that this mode requires the presence of the `shiftfs` kernel module, which
-currently is only available on the Ubuntu Linux distribution -- with the exception
-of Ubuntu-cloud images deployed in some Cloud Providers.
-
-### Directed userns ID mapping
-
-In this mode Sysbox relies on the container-manager selecting the desired ID
-mappings.
-
-In Docker's case this is accomplished by enabling the `userns-remap` feature
-which, even though imposes certain [restrictions](https://docs.docker.com/engine/security/userns-remap/#user-namespace-known-limitations),
-does not require `shiftfs` and therefore allows Sysbox support in a larger
-number of distributions.
-
-Refer to the Sysbox [installation guide](./user-guide/install.md#docker-userns-remap)
-for more details.
+-   "Kernel upgrade" means a kernel upgrade may be required (Sysbox-EE requires a fairly new kernel). See [below](#kernel-upgrade-procedures) for more.
```

Comment:
It's not that new anymore, 5.2 is more than 2 years old now, i would say "relatively new" :-)

Reply:
Done ...